Escape on 'tooltip' dialogs only works when the focus is on the initial input field org-ids/web-ide-issues#385

squarti opened the pull request.

squarti committed:
Escape on 'tooltip' dialogs only works when the focus is on the initial input field org-ids/web-ide-issues#385

carmacleod committed:
- close slideout on ESC if it has focus

carmacleod reviewed (commented):
Added code to close slideouts (Find, Go to line, ...) on Esc.

carmacleod reviewed (approved):
+1

carmacleod committed:
review:
- self is not defined
- call cancelFunction and closeFunction from getFillFunction

carmacleod committed:
- prefix with underscore for private fields

squarti committed:
review: reuse key handlers in fill function

squarti committed:
review: fix slide out panels

squarti committed:
review: fix filter commits panel

carmacleod reviewed (approved):
+1
